lib/docker.ts

```diff
@@ -41,12 +41,16 @@ class Docker {
     imageSavePath: string,
     username?: string,
     password?: string,
+    platform?: string | undefined,
```

Comment:
```suggestion
    platform?: string,
```

`undefined` is redundant here as the parameter is already marked as optional with `?`.